dmipy/core/modeling_framework.py

```diff
@@ -558,7 +558,8 @@ def _add_fixed_parameter_array(self, parameter_name, parameter_array):
     def set_tortuous_parameter(self, lambda_perp_parameter_name,
                                lambda_par_parameter_name,
                                volume_fraction_intra_parameter_name,
-                               volume_fraction_extra_parameter_name):
+                               volume_fraction_extra_parameter_name,
+                               S0_responses=None):
```

Comment:
Again (see below) I don't think we should be able to give S0 responses twice for any MC-model - it should be unique in the definition.

Reply:
Is there a shortcut for knowing which of the S0s in `self.S0_tissue_responses` list corresponds to the intra and which one to the extra?

Reply:
I didn't even thing about this when I first coded this... I especially kept the parameter names as input because I didn't want to have to hardcode the ordering or definitions of models to get the tortuosity regardless.

Actually we do need to find a solution here - what you were doing with adding the S0_responses as a parameter may be the easiest way to make this work after all... Either this or we need to define a nice way to just point to the models that are intra and extra, and then make the function just find which parameters and S0s are associated with this...